tests/unit/v1/testdata/query-invalid-operator.json

```diff
@@ -1,25 +0,0 @@
-{
-  "tests": [
-    {
-      "description": "query: invalid operator in Where clause",
-      "comment": "The !=  operator is not supported.",
-      "query": {
-        "collPath": "projects/projectID/databases/(default)/documents/C",
-        "clauses": [
-          {
-            "where": {
-              "path": {
-                "field": [
-                  "a"
-                ]
-              },
-              "op": "!=",
-              "jsonValue": "4"
-            }
-          }
-        ],
-        "isError": true
-      }
-    }
-  ]
-}
```

Comment:
The [upstream for this file](https://github.com/googleapis/conformance-tests/blob/master/firestore/v1/query-invalid-operator.json) is authoritative:  we are not supposed to modify / remove the copied-in files.

@BenWhitehead please clarify the intention here.

Reply:
PR to update the operator in the upstream files: https://github.com/googleapis/conformance-tests/pull/41

Reply:
Above PR has been merged and can be pulled into the project to test the actual case for a not supported operator.